style/main.css

```diff
@@ -0,0 +1,364 @@
+      display: inline-block;
+      padding: 1px 10px 30px 10px;
+      text-decoration: none;
+      color: #efefef;
+   }   
+  
+  .nav > .nav-links > a:hover {
+    background-color: rgba(0, 0, 0, 0.3);
+  }
+  
+  .nav > #nav-check {
+    display: none;
+  }
+
+  .main_features {
+   
+    background-color: #efefef;
+    padding-top: 10px;
+}
+
+.main_howto {
+    height: 35vh;
+    background-color: #efefef;
+}
+  
+.footer {
+    padding-top: 2px;
+    height: 7vh;
+    background-color:black;
+    color: wheat;
+}
+
+
+.feature{
+    width: 70vw;
+    box-shadow: 0 14px 28px rgba(0,0,0,0.25), 0 10px 10px rgba(0,0,0,0.22);
+margin-left: 2vw;
+padding-bottom: 20px;
+}
+.features {
+    display: flex;
+    width: 75vw;
+    margin: 0 auto;
+    /* text-align: center; */
+    padding-bottom: 50px;
+    height: ;
+} 
+.modal {
+    
+    position: fixed;
+    left: 0;
+    top: 0;
+    width: inherit;
+    height: 100%;
+    background-color: rgba(0, 0, 0, 0.5);
+    opacity: 0;
+    visibility: hidden;
+    transform: scale(1.1);
+    transition: visibility 0s linear 0.25s, opacity 0.25s 0s, transform 0.25s;
+}
+.modal2 {
+    
+    position: fixed;
+    left: 0;
+    top: 0;
+    width: inherit;
+    height: 100%;
+    background-color: rgba(0, 0, 0, 0.5);
+    opacity: 0;
+    visibility: hidden;
+    transform: scale(1.1);
+    transition: visibility 0s linear 0.25s, opacity 0.25s 0s, transform 0.25s;
+}
+.modal3 {
+    
+    position: fixed;
+    left: 0;
+    top: 0;
+    width: inherit;
+    height: 100%;
+    background-color: rgba(0, 0, 0, 0.5);
+    opacity: 0;
+    visibility: hidden;
+    transform: scale(1.1);
+    transition: visibility 0s linear 0.25s, opacity 0.25s 0s, transform 0.25s;
+}
+.modal-content {
+    padding: 2vh;
+    width: initial;
+    height: initial;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    margin: 30px auto;
+    background-color: white;
+    border-radius: 5px;
+    box-shadow: 10px 10px 40px 0px rgba(0,0,0,0.35);
+}
+
+.fitures{
+    display:flex;
+ font-size:1vh;
+ padding: 0; 
+    margin: 0 auto;
+    height: inherit;
+    
+}
+
+.fiture{
+    align-items: center;
+    display: flex;
+    margin: 0 auto;
+    justify-content: center;
+    /* text-align: center; */
+    height: 75%;
+    font-size: 2vh;
+    padding: 0;
+    WIDTH: 30vh;
+}
+
+.fiture:hover{
+    background-color:#699ece;
+}
+
+.modal-line {
+    width: 40vw;
+    border: 0.5vh;
+    margin:0;
+}
+
+.modal-header > h4{
+    margin: 2vh;
+}
+.close-button {
+    float: right;
+    width: 1rem;
+    line-height: 1rem;
+    text-align: center;
+    cursor: pointer;
+    border-radius: 0rem;
+    background-color: lightgray;
+}
+.close-button:hover {
+    background-color: darkgray;
+}
+.show-modal {
+    opacity: 1;
+    visibility: visible;
+    transform: scale(1.0);
+    transition: visibility 0s linear 0s, opacity 0.25s 0s, transform 0.25s;
+}
+
+.main_howto{
+
```

Comment:
Please remove this selector if not needed

Reply:
- done